### PR TITLE
SI-6897, lubs and varargs star.

### PR DESCRIPTION
**Pull request by paulp**
Don't allow lubs to calculate refinement types which contain
a varargs star outside of legal varargs star position.

### CODE REVIEW

**Comment by paulp** [comment]
PLS REBUILD ALL

File: src/reflect/scala/reflect/internal/Types.scala [hunk 1]
```diff
@@ -6801,7 +6801,10 @@ trait Types extends api.Types { self: SymbolTable =>
             else lubBase
           }
         }
-      existentialAbstraction(tparams, lubType)
+      // dropRepeatedParamType is a localized fix for SI-6897. We should probably
```
Comment (JamesIry) [inline src/reflect/scala/reflect/internal/Types.scala:6804]: Did you create an issue to track the change you want to do in master?

**Comment by JamesIry** [comment]
Other than comment about creating an issue to track the change you want to make in master, LGTM

File: src/reflect/scala/reflect/internal/Types.scala (hunk 1 above)
Comment (paulp) [inline src/reflect/scala/reflect/internal/Types.scala:6804]: I'll do it momentarily.

Comment (paulp) [inline src/reflect/scala/reflect/internal/Types.scala:6804]: https://issues.scala-lang.org/browse/SI-6929

**Pull request merged**